portfolio/src/main/webapp/script.js

```diff
@@ -14,9 +14,9 @@
 
 document.addEventListener('DOMContentLoaded', async function() {
   // Initialise the slider (index.html) and the material box (photoGallery.html)
-  elems_one = document.querySelectorAll('.slider');
+  const elems_one = document.querySelectorAll('.slider');
   instances_one = M.Slider.init(elems_one);
```

Comment:
could more variables without scope specifiers

Reply:
Oh yes, I missed those! I have made the changes